<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyImageFileClass</class>
 <widget class="QDialog" name="ModifyImageFileClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ModifyImageFile</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>ModifyImageFile</string>
     </property>
     <widget class="QCheckBox" name="checkBox_modifyFileFormat">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Modify File Format</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_modifyFileSize">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Modify file size</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Dest Folder:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_destFolder">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>250</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sourceFolder">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>30</y>
        <width>75</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Browser...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_destFolder">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>60</y>
        <width>75</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Browser...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_sourceFolder">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>250</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Source Folder:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Dest File Format:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Source File Format:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_execute">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>240</y>
        <width>75</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Execute</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>left:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>top:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>210</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>width:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>210</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>height:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_left">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>210</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_right">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>210</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_width">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_height">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>210</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_sourceFileFormat">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>220</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_destFileFormat">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>220</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>300</width>
        <height>20</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
